src/js/shield_canvas_draw.js

```diff
@@ -86,8 +86,15 @@ export function rectangle(ref) {
   );
 }
 
-export function blank() {
-  return Gfx.getGfxContext({ width: 1, height: 1 });
+export function blank(ref) {
```

Comment:
Parameter unused

Reply:
Are you sure? `ref` is right there on line 91.